### ISSUE TITLE
kubernetes-image-puller-operator 1.0.0 is in the CrashLoopBackOff State due to low memory limits

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che#21448]
### Describe the bug

![image](https://user-images.githubusercontent.com/1461122/172627927-f451376b-a109-425a-a50e-24878f505f3f.png)

### Che version

other (please specify in additional context)

### Steps to reproduce

Install 1.0.0 image puller on a cluster with 100+ namespaces
ERROR: Operator pod is crashing

### Expected behavior

Operator pod is running without errors

### Runtime

OpenShift

### Installation method

OperatorHub

### Environment

Amazon

### Additional context

Probably the limits are low:

```
Resource requests
cpu: 100m, memory: 64Mi
Resource limits
cpu: 100m, memory: 128Mi
```

**Comment by ibuziuk** [comment]
looks like current limits are not enough for large clusters. Installation failed on the developer sandbox. As a workaround the memory limit in csv has been updated to 1Gi

![image](https://user-images.githubusercontent.com/1461122/182191666-a8b81540-a98f-485e-901e-22968ad27a3a.png)

**Comment by ibuziuk** [comment]
@dkwon17 I believe we need to release 1.0.2 with more generous limits for operator

**Comment by ibuziuk** [comment]
According to the metrics we probably need to set it to 1Gi to be on the safe side. During the startup there is a spike, but then RAM consumption drops to ~ 400M

![image](https://user-images.githubusercontent.com/1461122/182356297-71947480-d300-4e80-b905-16ffc07dae84.png)

**Comment by dkwon17** [comment]
@ibuziuk I made a PR: https://github.com/che-incubator/kubernetes-image-puller-operator/pull/83

**Comment by ibuziuk** [comment]
Fixed in 1.0.2 version https://operatorhub.io/operator/kubernetes-imagepuller-operator 
Closing
